lib/lighthouse/letters_generator/client.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+require 'lighthouse/letters_generator/configuration'
+
+module Lighthouse
+  module LettersGenerator
+    def self.measure_time(msg)
+      start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+
+      response = yield
+
+      end_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      elapsed = end_time - start_time
+
+      Rails.logger.info "#{msg}: #{elapsed} seconds"
+      response
+    end
+
+    class Client < Common::Client::Base
```

Comment:
Should we change all of the services in lighthouse that we manage to be called `Client` instead of `Service` do you think? I'd like to maintain consistency

Reply:
I could go either way. Most everything is called `service` so I'll change it to conform